tango_ros_common/tango_ros_native/src/occupancy_grid_file_io.cpp

```diff
@@ -45,6 +85,13 @@ bool SaveOccupancyGridDataToPgmFile(
     const nav_msgs::OccupancyGrid& occupancy_grid) {
   std::string map_directory_with_trailing_slash = map_directory;
```

Comment:
I could not use const ref here because `AddTrailingSlashToDirectoryPathIfNeeded` had a non const reference argument. I changed it to use a pointer instead to avoid confusion.